### PR TITLE
Add postcss/cssnext support

### PR DESCRIPTION
**Pull request by paulathevalley**
Resolves https://github.com/FormidableLabs/builder-docs-archetype/issues/25 
- Add support with `postcss` and `postcss-cssnext`
- Use ExtractTextPlugin for the static build

### CODE REVIEW

**Comment by kylecesmat** [comment]
Looks great to me! 🚢

**Comment by paulathevalley** [comment]
Update: the `build-static` task is broken, but it may be due to `npm link`ing the archetype rather than the webpack configuration... will need to use `npm pack` to accurately test, I think.

**Comment by paulathevalley** [comment]
Ok, everything is working now!

File: config/webpack/webpack.config.static.js [hunk 1]
```diff
@@ -27,36 +30,72 @@ module.exports = {
...
-      compress: {
-        warnings: false
-      }
-    }),
+    // new webpack.optimize.DedupePlugin(),
+    // new webpack.optimize.UglifyJsPlugin({
+    //   compress: {
+    //     warnings: false
+    //   }
+    // }),
```
Comment (bmathews) [inline config/webpack/webpack.config.static.js]: intentionally removed?

Comment (paulathevalley) [reply config/webpack/webpack.config.static.js]: shoot! nope, I was debugging. will add back

**Review by bmathews — APPROVED** [review]

**Pull request merged**